Theme1/src/firsttheme/PrefixFinder.java

```diff
@@ -0,0 +1,19 @@
+package firsttheme;
+
+import java.util.Locale;
+
+public class PrefixFinder {
+    public static String findAPrefix(String[] strings) {
+        if(strings.length == 0) return "";
+        String result = strings[0].toLowerCase();
+        if (strings.length == 1) return result;
+
+        for(int i = 1; i < strings.length; i++) {
+            while (strings[i].toLowerCase().indexOf(result) != 0) {
```

Comment:
your solution has O(n^2) complexity because of nested loop.
Try to find solution for this task with O(n) complexity